_includes/corridors/usd-bank.md

```diff
@@ -18,8 +18,8 @@ For United States bank payments please use:
 ```
 {% endcapture %}
 
-{% include language-tabbar.html prefix="us-bank-details" raw=data-raw %}
+{% include language-tabbar.html prefix="usd-bank-details" raw=data-raw %}
```

Comment:
The reason this was setup as us-bank and not usd-bank is because `USD::Bank` serves both Nigeria and the United States. So with `us-bank` it's explicit that it refers to the United States. This should be reverted.

Reply:
Changes applied.